### ISSUE TITLE
[13444] View & Update ticket configurations within DD Tech.

### ISSUE THREAD

**Issue by Aycorn** [DeskDirector/UserVoice#252]
As an extension to feature request: https://github.com/DeskDirector/UserVoice/issues/51

Feature request to also be able to add/remove configurations against a ticket.

**Comment by Lowell-Syncrety** [comment]
Yes, this would be great. We require technicians to make sure they add the proper configuration(s) to each ticket before they close them.

**Comment by dr3dgen** [comment]
Agreed! This would be very helpful in our environment!

**Comment by appwjen** [comment]
adding this ticket #48427
